apps/src/utils.js

```diff
@@ -695,3 +695,16 @@ export function navigateToHref(href) {
     window.location.href = href;
   }
 }
+
+/**
+ * Resets the animation of an aniGif by unsetting and setting the src
+ * @param {Element} el the <img> element that needs to be reset
+ */
+export function resetAniGif(el) {
+  if (!el) {
+    return;
+  }
+  const src = el.src;
+  el.src = '#';
+  setTimeout(() => {el.src = src;}, 0);
```

Comment:
nit: i think braces are unnecessary here (and if you have them i would prefer newlines). i.e. you can do `setTimeout(() => el.src = src, 0);`

Reply:
have we checked that this works in all browsers? for example, i know FF handles anigifs differently than chrome in some respects

Reply:
Removed the braces. It works in FF, but I couldn't get it working on IE/Edge :disappointed: